src/main/java/com/yelp/nrtsearch/server/luceneserver/field/DateTimeFieldDef.java

```diff
@@ -55,6 +55,8 @@
 public class DateTimeFieldDef extends IndexableFieldDef implements Sortable, RangeQueryable {
   private static final long MIN_DATE_TIME = 0;
 
+  private static final String EPOCH_MILLIS = "epoch_millis";
```

Comment:
It would be helpful to have it mentioned in https://github.com/Yelp/nrtsearch/blob/master/clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto#L419 that the format can be "epoch_millis" too apart from formats like "dd-mm-yyyy"